project-base/docker/php-fpm/docker-php-consumer-entrypoint

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+default_tries_before_fail=100
+messages_to_consume_count=1000
+
+consumer_name=$1
+
+if [[ -z ${consumer_name} ]]; then
+    1>&2 echo "First argument had to be name of consumer to start"
+    exit 1
+fi
+
+if [[ -n ${CONSUMER_RUNS_BEFORE_RESTART} ]]; then
```

Comment:
You can simplify this into a oneliner:
```bash
messages_to_consume_count=${CONSUMER_RUNS_BEFORE_RESTART:-$messages_to_consume_count}
```

I would even use it in the first initialization, removing thus block altogether:
```bash
messages_to_consume_count=${CONSUMER_RUNS_BEFORE_RESTART:-1000}
```

For more tips on parameter expansions in Bash I recommend [devhints.io](https://devhints.io/bash#parameter-expansions).

Reply:
Nice! Thanks for the link.

Fixed in 4ce19148f594490a1c304de651f71aa85e2098e8

Reply:
It's a bit weird for me to have the `CONSUMER_RUNS_BEFORE_RESTART` environment variable defined in all consumer services in `docker-compose.yml` files (and use the default value of `1000`).

Can't we have the env var explained in the docs instead of cluttering the configuration?

Reply:
It will be better, you're right.

fixed in b08763dd9237cd8da7c0f03c4769427d47c6a4bc 81ce57cdfc6b1535abe061c15bed9f9f0f44b631 and 14f11f7d5dc307eb361fc1b8289a43ea43be01af

Reply:
It will be better, you're right.

fixed in
b08763dd9237cd8da7c0f03c4769427d47c6a4bc 81ce57cdfc6b1535abe061c15bed9f9f0f44b631
14f11f7d5dc307eb361fc1b8289a43ea43be01af

Reply:
Great :+1: You've missed one occurrence in the upgrade notes, so I fixed it in 5d5634ad5366e4538e3e5deff54d445c4133f806

Reply:
Thanks!